odml/tools/format_converter.py

```diff
@@ -119,6 +119,9 @@ def _convert_file(cls, input_path, output_path, res_format):
                 p, _ = os.path.splitext(output_path)
                 output_path = p + cls._conversion_formats[res_format]
             RDFWriter(odml.load(input_path)).write_file(output_path, res_format)
+        else:
+            raise ValueError("Format for output files is incorrect. "
+                             "Please choose from the list: {}".format(cls._conversion_formats.keys()))
```

Comment:
In Python 3 this will print `dict_keys(....)`. Maybe using `list(cls._conversion_formats.keys())` is a better option here?

Reply:
To go one step further, if using `list(cls._conversion_formats)` should print just the keys as a list in both py2 and py3.